Content.Server/Mousetrap/MousetrapSystem.cs

```diff
@@ -61,6 +69,65 @@ private void OnTrigger(EntityUid uid, MousetrapComponent component, TriggerEvent
         UpdateVisuals(uid);
     }
 
+    private void AddMouseDefuseVerb(EntityUid uid, MousetrapComponent component, GetVerbsEvent<Verb> args)
+    {
+        if (!args.CanInteract
+            || !args.CanAccess)
+            return;
+
+        if (!TryComp(uid, out DamageUserOnTriggerComponent? damageOnTrigger))
+            return;
+
+        Verb defuse = new()
+        {
+            Act = () =>
+            {
+                if (!component.IsActive)
+                {
+                    _popupSystem.PopupEntity(Loc.GetString("mousetrap-is-not-activated"), uid, args.User);
+                    return;
+                }
+
+                _audio.PlayPvs(Comp<EmitSoundOnTriggerComponent>(uid).Sound, uid); // play snAp
+
+                if (_random.Prob(MathF.Min(0.99f, 0.15f * component.Difficulty))) // Chance to die
```

Comment:
Don't use a dummy min statement, just check the value.

Also having a random chance to die makes this feature nearly pointless when they can just go around the mousetrap 99% of the time.